### DISCUSSION TITLE
Welcome to Contributing-guide-generator Discussions!

### DISCUSSION THREAD

**Discussion by Haimantika** [Haimantika/Contributing-guide-generator#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shyamtawli** [comment]
Can we migrate it to ReactJS or NextJS?

**Reply by Haimantika** [reply]
NextJS is what I am aligning towards. What do you think?

**Reply by shyamtawli** [reply]
Yeah that would be great. NextJS can work as full stack, it will be easy for backend too.
